pkgs/development/interpreters/evcxr/default.nix

```diff
@@ -0,0 +1,28 @@
+{ cargo, fetchFromGitHub, makeWrapper, pkgconfig, rustPlatform, stdenv, zeromq }:
+
+rustPlatform.buildRustPackage rec {
+  name = "evcxr-${version}";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "evcxr";
+    rev = "ae07ccf08723b7aec0de57d540822b89088ca036";
+    sha256 = "1apc93z9vvf6qks5x2pad45rnrj9kjl812rj78w5zmmizccp2fhf";
+  };
+
+  cargoSha256 = "1pvw1jfi4p0cla4xxzgxh1jskvbmsyi4l5cimiis7093hyjs0svw";
+  cargoPatches = [ ./cargo-lock.patch ];
+
+  nativeBuildInputs = [ pkgconfig zeromq ];
+  buildInputs = [ cargo makeWrapper ];
```

Comment:
makeWrapper should be a `nativeBuildInput`.

Reply:
Indeed. Fixed.